### PR TITLE
Added pre-commit hook

### PR DESCRIPTION
**Pull request by ternaus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,23 +218,25 @@ Python and library versions: Python 3.7.3 | Anaconda, numpy 1.16.4, pillow 6.0.0
...
+4. Modify it.
+5. Install the library in development mode:
+```
+pip install -e .[tests]
+```
+6. Run tests:
+```
+pytest
+```
+7. Push code to your forked repo.
```
Comment (albu) [inline README.md]: flake8?

File: README.md [hunk 2]
```diff
@@ -218,23 +218,25 @@ Python and library versions: Python 3.7.3 | Anaconda, numpy 1.16.4, pillow 6.0.0
...
-   ```
+
+1. Fork the repository:
+2. Clone it.
+3. Install pre-commit hook:
+```
+pip install pre-commit
+pre-commit install
+```
+4. Modify it.
```
Comment (albu) [inline README.md]: modify pre-commit hook? how?

File: README.md (hunk 1 above)
Comment (ternaus) [reply README.md]: Automatically evaluated by a pre-commit hook.

File: README.md (hunk 2 above)
Comment (ternaus) [reply README.md]: I meant, make desired modification in the code. Not in the pre-commit hook.

Rephrased.

File: README.md [hunk 3]
```diff
@@ -218,23 +218,25 @@ Python and library versions: Python 3.7.3 | Anaconda, numpy 1.16.4, pillow 6.0.0
...
-   ```
-   flake8
-   ```
+
+1. Fork the repository:
+2. Clone it.
+3. Install pre-commit hook:
+```
+pip install pre-commit
+pre-commit install
```
[the remark this answers is not in the document]
Comment (ternaus) [reply README.md]: Fixed.

**Comment by ternaus** [comment]
1. Added Pre-commit hook for black.
2. Moved text with contributions to contributions.rst
3. Fixed First/Last names in docs.
4. Pre-commit hook was enabled while [3] => conf.py was updated to black standards.

File: README.md [hunk 4]
```diff
@@ -218,23 +218,8 @@ Python and library versions: Python 3.7.3 | Anaconda, numpy 1.16.4, pillow 6.0.0
...
-3. Run tests:
-   ```
-   pytest
-   ```
-4. Run flake8 to perform PEP8 and PEP257 style checks and to check code for lint errors.
-   ```
-   flake8
-   ```
+
+To create pull request to the repsitory follow the the documentation at [docs/contributing.rst](docs/contributing.rst)
```
Comment (Dipet) [inline README.md]: Please fix "follow the the"

Comment (ternaus) [reply README.md]: Thanks!

**Pull request merged**